iib/workers/tasks/opm_operations.py

```diff
@@ -907,3 +907,224 @@ def verify_operator_exists(
             is_package_in_index = True
             log.info("operator package %s found in index_db %s", operator_package, index_db_path)
     return is_package_in_index, index_db_path
+
+
+def set_opm_verison(index_image: str) -> str:
+    """
+    This function will verify if opm config is set in worker
+    if the config entry exists for the index image version
+    and return the opm version path to be used in the entire IIB request.
+    :param str index_image: from_index_image for the request
+    :return str opm_path: opm_path to be used for the entire request
+
+    """
+    global opm_path
+    log.info("Determining wthe OPM version to use")
+    worker_config = get_worker_config()
+    opm_paths_config = worker_config.get("iib_ocp_opm_mapping")
+    if not opm_paths_config:
+            log.error("OPM config is not set")
```

Comment:
Is it really an error? I think this could be backwards compatible. Therefore I would just logged info message that iib_ocp_opm_mapping is not set. Unless we want to make it mandatory.

Reply:
Should net be the message something like "iib_ocp_opm_mapping configuration is not set"

Reply:
When `opm_paths_config` is not set I think we should call `return` and do not continue.

Reply:
I am not sure whether it should be mandatory but we definitely need to support multiple opm in IIB soon

Reply:
updated already